wordle-web/scripts/wordOptions.ts

```diff
@@ -0,0 +1,37 @@
+import { WordsService } from '~/scripts/wordsService'
+
+ export class wordOptions {
+   words: string[] = []
+   count: number = 0
+
+   validWords(word: string): string[] {
+     this.words = []
+     if (!word.includes('?')) {
+       this.words.push('')
+       this.count = 0
+       return this.words
+     }
+
+     let expression: string = '^[A-Z]{0,5}'
+     for (let i = 0; i < 5; i++) {
```

Comment:
your regex might also be a problem.  Regex is case sensitive, and this is looking for only capital letters, but if you look at the list of words we are checking against, the words are all in lower case letters. Unless somewhere you are using an ignore case and I'm not seeing it.